### ISSUE TITLE
Pass name of changed file as argument to command

### ISSUE THREAD

**Issue by vithar** [kimmobrunfeldt/chokidar-cli#11]
I want to know which file is changed in my command in order to run build proccess for only one changed file, not all files in watched directory.

Can you pass names of changed files as arguments to command being executed?

E.g. I have script

test.sh:
echo *

I run chokidar:

chokidar 'mydir/**' -c './test.sh'

When I change file `afile` in `mydir` — `test.sh` is executed with argument `afile`

**Comment by kimmobrunfeldt** [comment]
It would be convenient if there would be an environment variable to contain the path of changed file/folder as @serapath mentioned in the other issue thread.

In the meanwhile, you have to do command line plumbing to get the path of changed file. I don't have a direct example of how to do that, but you can get changed file paths to stdout like this:

```
$ chokidar '**/.js' '**/*.less' | cut -d ':' -f 2-
test/dir/a.js
test/dir/a.less
test/b.js
test/b.js
```

**Comment by kimmobrunfeldt** [comment]
Fix for this released now in 1.2.0: https://github.com/kimmobrunfeldt/chokidar-cli/releases/tag/1.2.0
